phobos_tests_extractor.d

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env dub
+/+ dub.sdl:
+name "check_phobos"
+dependency "libdparse" version="~>0.7.0-beta.2"
++/
+/*
+ * Parses all public unittests that are visible on dlang.org
+ * (= annotated with three slashes)
+ *
+ * Copyright (C) 2016 by D Language Foundation
+ *
+ * Distributed under the Boost Software License, Version 1.0.
+ *    (See accompanying file LICENSE_1_0.txt or copy at
+ *          http://www.boost.org/LICENSE_1_0.txt)
+*/
+// Written in the D programming language.
+
+import std.stdio;
+import std.conv;
+import std.string;
+import std.file;
+import std.regex;
+import std.range;
+import std.exception;
+import std.algorithm;
+import std.path;
```

Comment:
On thing I'd like to add to our usual style guidelines is imports should be ordered alphabetically (it did happen at least to me to import something twice when the imports list is long). Also, I wonder how our opinionated community would feel about preferring comma-separated imports instead of one per line.